internal/plugin.go

```diff
@@ -386,10 +386,26 @@ func (p *Plugin) applyDefaults(unmarshaledConfig map[string]interface{}) {
 
 	for i := range p.Policies {
 		policy := &p.Policies[i]
+		if len(policy.PolicyAnnotations) == 0 {
```

Comment:
The issue with this approach is that I can't overwrite the policy default's annotations when setting an empty map at the policy level.

I'm not 100% sure, but I think if you change this to `if policy.PolicyAnnotations == nil {` it would work. Could you please try that?